src/test/java/io/confluent/connect/hdfs/wal/WALFileTest.java

```diff
@@ -36,10 +36,12 @@
   @Test
   public void testAppend() throws Exception {
     setUp();
+    properties.put(HdfsSinkConnectorConfig.TOPIC_REGEX_CAPTURE_GROUP_CONFIG, "(.*)");
```

Comment:
why is `testAppend` special?

Reply:
it uses a different topic that isn't captured by the normal regex